geometry_msgs/PoseStamped pose_stamped
std_msgs/String tasksrv
std_msgs/String tasksubtree